common/test/acceptance/tests/studio/test_studio_settings.py

```diff
@@ -581,9 +580,7 @@ def test_special_exams_menu_a11y(self):
             include=['section.edit-settings-timed-examination']
         )
         self.course_outline.a11y_audit.config.set_rules({
-            "ignore": [
-                'section',  # TODO: AC-491
-            ]
+            "ignore": []
```

Comment:
Nit: if we aren't ignoring anything, I'd suggest just deleting the set_rules call.